cmd/vic-machine/create/create.go

```diff
@@ -176,6 +198,166 @@ func (c *Create) Flags() []cli.Flag {
 	return flags
 }
 
+func zeroIPNet(n net.IPNet) bool {
+	return n.IP == nil && n.Mask == nil
+}
+
+func parseGatewaySpec(cg string) (string, *net.IPNet, error) {
+	if cg == "" {
+		return "", nil, cli.NewExitError("container network gateway is empty", 1)
+	}
+
+	mapped := strings.Split(cg, ":")
+	if len(mapped) != 2 {
+		return "", nil, cli.NewExitError(fmt.Sprintf("Container network gateway %s not in VSPHERE-NETWORK:GATEWAY format", cg), 1)
```

Comment:
Worth noting that gateway is IPv4 here otherwise people will wonder why their IPv6 is causing this message.